microsoft/botframework-solutions
# [Skill Template][TypeScript] Implement tests and fixes

## Description
- Implement standalone functionality from `SampleSkill`
- Remove placeholders for `skillConversationState` for parity with C#
- Add placeholders to `cognitiveModels` files
- Change `i18n` to `i18next`
- Added tests and mocks

## Testing Steps
1. Install Yeoman `npm install -g yo`.
1. Go to `.\templates\Skill-Template\src\typescript\generator-botbuilder-skill`, run `npm i` and `npm link` to symlink the package folder.
1. **[OPTIONAL]** Use the command `npm run test` to run the Generator's tests.
1. Open a terminal wherever you want to create a skill.
1. Run `yo botbuilder-skill`.
1. Follow the generator's prompts.
1. Check that the skill was created successfully **without placeholders**.
1. **[OPTIONAL]** Use the command `npm run test` to run the Skill's tests.

## Checklist
